### DISCUSSION TITLE
The Compatibility

### DISCUSSION THREAD

**Discussion by lightyagamiklmn** [skyline-emu/skyline#1068] [General]
So, I don't know much about coding and emulation development, that is why there is one thing I do not understand.
When you work on a game to make it compatible with the emulator, is it SPECIFICALLY for that game only or does making that certain game compatible affect the compatibility of some other games too?
For example: When you work on, say, TLOZBOTW and you end up making it compatible/playable, will that indirectly affect some other games with similar stuff or does the other game have to be worked on from the start again?
Sorry if I bothered you with this question, I was just really curious about this.
Thank you for reading!

**Comment by PixelyIon** [comment]
The latter, working on one game just helps us organize how we work but it affects a multitude of games to implement something for a specific game. This is why even though we're building up to SMO so many other games are booting in the process.

**Reply by lightyagamiklmn** [reply]
Ahh I see, I understand.
Thank you for clearing it up!
